keel-sql/src/main/resources/jooqConfig.xml

```diff
@@ -14,6 +14,24 @@
         DATABASECHANGELOG
         | DATABASECHANGELOGLOCK
       </excludes>
+      <forcedTypes>
+        <!-- Automatically convert JSON columns to String -->
+        <forcedType>
+          <name>LONGVARCHAR</name>
+          <includeTypes>JSON</includeTypes>
+        </forcedType>
+        <!-- Automatically convert TIMESTAMP to Instant -->
+<!--        <forcedType>-->
+<!--          <name>INSTANT</name>-->
+<!--          <includeTypes>TIMESTAMP.*</includeTypes>-->
+<!--        </forcedType>-->
+        <!-- Convert LocalDateTime to Instant -->
+        <forcedType>
+          <includeTypes>TIMESTAMP.*</includeTypes>
+          <userType>java.time.Instant</userType>
+          <converter>com.netflix.spinnaker.keel.sql.JooqTimestampsToInstantConverter</converter>
```

Comment:
This works a bit better, except that JOOQ gets lost with the types in certain cases and tries to pass a `java.sql.Timestamp` to the converter instead of a `java.time.LocalDateTime` as declared, which causes errors like these:
```
org.jooq.exception.DataAccessException: SQL [select 'keel' as `deliveryConfigName`, 'staging' as `environmentName`, `keel`.`environment_artifact_constraint`.`artifact_version`, `keel`.`environment_artifact_constraint`.`type`, `keel`.`environment_artifact_constraint`.`status`, `keel`.`environment_artifact_constraint`.`created_at`, `keel`.`environment_artifact_constraint`.`judged_by`, `keel`.`environment_artifact_constraint`.`judged_at`, `keel`.`environment_artifact_constraint`.`comment`, `keel`.`environment_artifact_constraint`.`attributes` from `keel`.`environment_artifact_constraint` where `keel`.`environment_artifact_constraint`.`environment_uid` = ? order by `keel`.`environment_artifact_constraint`.`created_at` desc limit ?]; Error while reading field: "keel"."environment_artifact_constraint"."created_at", at JDBC index: 6
	at org.jooq_3.12.3.MYSQL.debug(Unknown Source)
	... 
Caused by: java.lang.ClassCastException: java.sql.Timestamp cannot be cast to java.time.LocalDateTime
	at com.netflix.spinnaker.keel.sql.JooqTimestampsToInstantConverter.from(JooqConverters.kt:11)
	... 
```
😭